ipatests/test_webui/ui_driver.py

```diff
@@ -2369,3 +2406,14 @@ def assert_value_checked(self, values, name, negative=False):
             else:
                 assert value in checked_values, ('{} NOT checked while it '
                                                  'should be'.format(value))
+
+    def assert_user_auth_type(self, auth_type, enabled=True):
```

Comment:
Could you please keep the method in **user_tasks** class, since it is only user related?

Reply:
Good point, moved